run_bot_streaming.sh

```
@@ -0,0 +1,283 @@
+    DOCKER_WEBSOCKET_URL="$WEBSOCKET_URL"
+    if [[ "$WEBSOCKET_URL" =~ localhost ]] || [[ "$WEBSOCKET_URL" =~ 127\.0\.0\.1 ]]; then
+        DOCKER_WEBSOCKET_URL=$(echo "$WEBSOCKET_URL" | sed 's/localhost/host.docker.internal/g' | sed 's/127\.0\.0\.1/host.docker.internal/g')
+        print_info "🔄 Converted WebSocket URL for Docker: $DOCKER_WEBSOCKET_URL"
+    fi
+
+    print_success "🚀 Starting bot with streaming to $DOCKER_WEBSOCKET_URL"
+    echo ""
+
+    # Run the bot with streaming enabled
+    echo "$processed_config" | docker run -i \
+        $docker_args \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)